docs/installguide/install_all.rst

```diff
@@ -0,0 +1,307 @@
+================
+
+1. Download the KA Lite `OSX installer <https://disney.com>`_.
+
+.. warning:: Dear maintainers, please update this link.
+
+Linux: Debian/Ubuntu Installation
+=================================
+
+Option 1, Ubuntu: Add a PPA
+___________________________
+
+We maintain a `PPA on Launchpad <https://launchpad.net/~learningequality/+archive/ubuntu/ka-lite>`_
+and if you are connected to the internet, this will also give you automatic updates::
+
+    sudo add-apt-repository ppa:learningequality/ka-lite
+    sudo apt-get update
+    sudo apt-get install ka-lite
+
+
+.. _linux-deb-install:
+Option 2, Ubuntu / Debian, download a Debian package
+____________________________________________________
+
+Download the latest .deb manually from
+`the Launchpad archive server <http://ppa.launchpad.net/learningequality/ka-lite/ubuntu/pool/main/k/ka-lite-source/?C=M;O=D>`_.
+Look for the latest file with a ".deb" extension, download it and open it with
+Ubuntu Software Center, GDebi, Synaptic, or install it from command line
+with ``dpkg -i  ka-lite_0.x.x-0ubuntu1_all.deb``.
+
+The file may be named as if it was intended for Ubuntu but works just as well
+for any other Debian-based systems like Debian, Raspberry Pi, Linux Mint etc.
+
+Configuration after installing / updating
+_________________________________________
+
+Every time you install or update kalite, you should (re)run ``kalite manage setup``
+to setup the database and download assessment items (video descriptions,
+exercises etc.).
+
+
+.. _linux-pypi-install:
+Linux: Installing as a PyPI package
+===================================
+
+To install KA Lite from a file, go to our `PyPi page <https://pypi.python.org/pypi/ka-lite-static>`_ and download a .tar.gz or .zip. You can then install the file from a system by unpacking it and running *python setup.py install* or *pip install ka-lite-static-X.Y.Z.tar.gz*.
+
+Alternatively, install with *pip* using the instructions below:
+
+.. note:: Type the following commands in a terminal.
+
+#. Install pip (Ubuntu: ``sudo apt-get install python-pip``)
+#. (Recommended; essential on slower platforms like Raspberry Pi) Install M2Crypto (``sudo apt-get install python-m2crypto``).
+#. Run ``sudo pip install ka-lite-static`` (bundled dependencies) or ``sudo pip install ka-lite`` (normal pip behaviour, dependencies installed to host system).
+#. Run ``kalite manage setup``.
+
+For more tips see :ref:`advanced-installation`.
+
+Configuration after installing / updating
+_________________________________________
+
+Every time you install or update kalite, you should (re)run ``kalite manage setup``
+to setup the database and download assessment items (video descriptions,
+exercises etc.).
+
+
+Uninstalling
+============
+
+You can remove KA Lite (when installed from pip or source distribution) with
+`pip uninstall ka-lite` or `pip uninstall ka-lite-static` (static version).
+
+For Ubuntu, use ``apt-get remove ka-lite``.
+
+
+Removing user data

```

Comment:
Also OS specific. `~` at least is not conventional on Windows, but the Python docs [describe how it is resolved](https://docs.python.org/2/library/os.path.html#os.path.expanduser).